README.md

```diff
@@ -1,21 +1,48 @@
-# 🐾 autokitteh
+# 🐾 AutoKitteh - VSCode Extension
```

Comment:
```suggestion
# 🐾 AutoKitteh - VS Code Extension
```

Reply:
Changed to "Visual Studio Code Extension"